tests/plugins/test_integralgrader.py

```diff
@@ -589,7 +590,7 @@ def test_debug_message_complex_integrand():
         "==============================================<br/>\n"
         "Integration Data for Sample Number 0<br/>\n"
         "==============================================<br/>\n"
-        "Variables: {{'infty': inf, 'e': 2.718281828459045, 'i': 1j, 'j': 1j, 's': 1.8831785881043805, 'x': 0.022981166359782736, 'pi': 3.141592653589793}}<br/>\n"
+        "Variables: {{{u}'infty': inf, {u}'e': 2.718281828459045, {u}'i': 1j, {u}'j': 1j, {u}'s': 1.8831785881043805, {u}'x': 0.022981166359782736, {u}'pi': 3.141592653589793}}<br/>\n"
```

Comment:
But apparently not here. (This suggests to me that you can just use `u` on all of these, instead of needing the prefix variable?)

Reply:
The `u` instead of `{u}` was probably a mistake. They are equivalent in Python 2, and we're not really running the Python 3 tests yet. (A bunch of them fail for completely unrelated versions). 

So my guess is that this test would fail under Python 3. 

BTW: I am going to be gone for about 4-5 hours starting now-ish, so I'll just let you finish the review :)w

Reply:
Ok, sounds good. Will let you update the missing entries then.

Reply:
changed to {u}